platforms/messenger/messenger.js

```diff
@@ -135,111 +136,111 @@ function receivedAccountLink (event) {
  * Send an image using the Send API.
  *
  */
-function sendImageMessage (recipientId) {
-  const messageData = {
-    recipient: {
-      id: recipientId
-    },
-    message: {
-      attachment: {
-        type: "image",
-        payload: {
-          url: `${SERVER_URL}/assets/rift.png`
-        }
-      }
-    }
-  };
-
-  callSendAPI(messageData);
-}
+// function sendImageMessage (recipientId) {
```

Comment:
IMHO, it will be useful for the next one, when more functions of the messenger API will be used. They are not in use for the moment, but can be useful in the future. So, I left them 😉👊